kahuna/public/js/crop/view.html

```diff
@@ -1,28 +1,29 @@
 <gr-top-bar>
     <gr-top-bar-nav>
         <a class="top-bar-item"
-           ui:sref="image({ imageId: ctrl.image.data.id })">← cancel</a>
+           ui:sref="image({ imageId: ctrl.image.data.id })"><i class="material-icons">cancel</i>
+           <span class="top-bar-item--label-text"> cancel</span></a>
     </gr-top-bar-nav>
 
     <gr-top-bar-actions>
-        <div class="top-bar-item">{{ ctrl.cropSize() }}</div>
+        <div class="top-bar-item top-bar-item--static top-bar-item--surplus">{{ ctrl.cropSize() }}</div>
 
-        <label class="top-bar-item">
-            <input type="radio" ng:model="ctrl.aspect" value="{{ctrl.landscapeRatio}}" />
-            landscape ({{::ctrl.getRatioString(ctrl.landscapeRatio)}})
+        <label class="top-bar-item" title="landscape">
+            <input type="radio" ng:model="ctrl.aspect" value="{{ctrl.landscapeRatio}}"/>
+            <span class="top-bar-item--label-text">landscape </span><i class="material-icons">crop_landscape</i> ({{::ctrl.getRatioString(ctrl.landscapeRatio)}})
         </label>
 
-        <label class="top-bar-item">
+        <label class="top-bar-item" title="portrait">
             <input type="radio" ng:model="ctrl.aspect" value="{{ctrl.portraitRatio}}" />
-            portrait ({{::ctrl.getRatioString(ctrl.portraitRatio)}})
+            <span class="top-bar-item--label-text">portrait </span><i class="material-icons">crop_portrait</i> ({{::ctrl.getRatioString(ctrl.portraitRatio)}})
```

Comment:
If we're using BEM - `top-bar-item__label` or `top-bar-label` would make more sense.